packages/lib/server-only/document/complete-document-with-token.ts

```diff
@@ -88,7 +88,20 @@ export const completeDocumentWithToken = async ({
       id: document.id,
       Recipient: {
         every: {
-          signingStatus: SigningStatus.SIGNED,
+          OR: [
```

Comment:
Is there a way we can simplify this query? Perhaps there isn't but it'd be interesting to find out 🙌🏻

Reply:
I've thought about it, for those that don't need to do anything with a document (CC'er) we can mark them as signed from the beginning.

For viewers I'm unsure, we may want them to confirm viewing rather than having simply opened the document much like an approver needs to confirm approving.